R/util.R

```diff
@@ -925,3 +925,13 @@ lock_bindings <- function(syms, env) {
   }
   invisible(NULL)
 }
+
+clean_report_name <- function(name) {
```

Comment:
This duplicates logic in the increasingly misnamed `orderly:::orderly_develop_location`; can we use that instead?

Reply:
This PR pulls that file name cleaning logic out of `orderly_develop_location` into this new function. Then `orderly_develop_location` calls `clean_report_name`. Or you think `orderly_ cleanup` should call `orderly_develop_location`?

Reply:
Sorry, my mistake - you're right of course